lib/watir-webdriver/browser.rb

```diff
@@ -41,7 +41,7 @@ def start(url, browser = :firefox, *args)
 
     def initialize(browser = :firefox, *args)
       case browser
-      when Symbol, String
+      when ::Symbol, String
```

Comment:
`Symbol` now resolves to `Watir::Symbol`, not `Object::Symbol`. That's why I added `::`.